model-optimizer/extensions/front/tf/mvn_unrolled.py

```diff
@@ -76,12 +77,10 @@ def infer(node: Node):
             log.warning('Reduction indices for mean and variance for MVN node {} are not constants'.format(node.name))
             return
 
-        if not (all(node.in_node(1).value == node.required_reduction_indices) and
-                    all(node.in_node(2).value == node.required_reduction_indices)):
-            log.warning('Reduction indices for mean {} and variance {} do not match required ones {}'.format(
+        if not (all(node.in_node(1).value == node.in_node(2).value)):
+            log.warning('Reduction indices for mean {} and variance {} do not match'.format(
                 node.in_node(1).value,
-                node.in_node(2).value,
-                node.required_reduction_indices
+                node.in_node(2).value
             ))
```

Comment:
Please, get rid of this ugly old-style approach with run-time modified `infer` method. The easiest solution is to move this transformation to the back phase where we can perform all necessary checks in the transformation itself. Later, we can implement the fusing transformation in nGraph and remove it from the MO.

Reply:
It is hard to do without knowing for what model it was done. The operations in the reference graph will change by the time of back phase.